core/domain/question_services_test.py

```diff
@@ -870,18 +964,12 @@ def test_migrate_question_state_from_v30_to_v31(self):
             'user_id_admin', 'question model created', commit_cmd_dicts)
 
         current_schema_version_swap = self.swap(
-            feconf, 'CURRENT_STATE_SCHEMA_VERSION', 31)
+            feconf, 'CURRENT_STATE_SCHEMA_VERSION', 33)
 
         with current_schema_version_swap:
             question = question_fetchers.get_question_from_model(question_model)
 
-        self.assertEqual(question.question_state_data_schema_version, 31)
-        self.assertEqual(question.question_state_data
-                         .recorded_voiceovers.to_dict(), {
-                             'voiceovers_mapping': {
-                                 'content': {
-                                     'en': {
-                                         'filename': 'test.mp3',
-                                         'file_size_bytes': 100,
-                                         'needs_update': False,
-                                         'duration_secs': 0.0}}}})
+        self.assertEqual(question.question_state_data_schema_version, 33)
+
+        cust_args = question.question_state_data.interaction.customization_args
+        self.assertEqual(cust_args['showChoicesInShuffledOrder']['value'], True)
```

Comment:
Should also check that all other aspects of cust_args are preserved.

Reply:
Done.